react/forms/step3.md

```diff
@@ -0,0 +1,68 @@
+Now we'll modify our form to include a title dropdown.
+
+## Task
+
+1) Make sure the App.js file is still open
+
+2) To create a title dropdown, we'll add a new `value1` attribute to state and a `this.options` array in the constructor.
+

```

Comment:
Can we please show the user what / where the constructor is? Some folks may not be familiar with that.

For example:

2) Locate the code that looks like the following:

```
constructor(props) {
...
}
```